apps/researcher/src/messages/en/about.mdx

```diff
@@ -0,0 +1,27 @@
+# About the Dataset Browser
```

Comment:
`About the Research Application`? But I guess Camiel can update this :)

Reply:
Yes. Currently, the text is also "About the Research Application" for both apps. I decided not to change any text in this pull request.